build.sbt

```diff
@@ -8,10 +8,12 @@ homepage := Some(url("http://github.com/non/kind-projector"))
 scalaVersion := "2.11.7"
 crossScalaVersions := Seq("2.10.5", "2.11.7", "2.12.0-M3")
 
-libraryDependencies <++= (scalaVersion) {
-  v => Seq("org.scala-lang" % "scala-compiler" % v)
+libraryDependencies <++= (scalaVersion) { v =>
+  Seq("org.scala-lang" % "scala-compiler" % v)
 }
```

Comment:
You might prefer:

    libraryDependencies += "org.scala-lang" % "scala-compiler" % scalaVersion.value